desktop/core/src/desktop/js/reactComponents/FileUploadQueue/FileUploadQueue.tsx

```diff
@@ -153,11 +157,28 @@ const FileUploadQueue = (): JSX.Element => {
     return result;
   };
 
-  const onClose = () => {
-    uploadQueue.forEach(file => cancelFile(file));
-    setIsVisible(false);
+  const handleFileUploadStart = async (data?: FileUploadEvent) => {
+    const newFiles = data?.files ?? [];
+    if (newFiles.length === 0) {
+      huePubSub.publish('hue.global.error', { message: 'Something went wrong!' });
```

Comment:
This error message needs to be improved. "Something went wrong!" doesn't add any value to our users.